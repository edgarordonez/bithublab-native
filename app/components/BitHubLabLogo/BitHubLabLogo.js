// @flow

import React, { Fragment } from 'react';
import { StyleSheet, Text, Image } from 'react-native';
import ASSETS from '@bithublab/assets';

export const BitHubLabLogo = () => (
  <Fragment>
    <Image
      source={ASSETS.images.logo}
      style={{ width: 150, height: 150 }}
      resizeMode="contain"
    />
    <Text style={styles.title}>
      <Text style={{ color: '#2684ff' }}>Bit</Text>
      <Text style={{ color: '#000000' }}>Hub</Text>
      <Text style={{ color: '#8c929d' }}>Lab</Text>
    </Text>
  </Fragment>
);

const styles = StyleSheet.create({
  title: {
    fontFamily: 'Roboto-Black',
    fontSize: 50,
  },
});
