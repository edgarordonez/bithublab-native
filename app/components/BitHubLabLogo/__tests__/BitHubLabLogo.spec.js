// @noflow

import React from 'react';
import TestRenderer from 'react-test-renderer';

import { BitHubLabLogo } from '../BitHubLabLogo';

describe('BitHubLabLogo', () => {
  it('Renders without crashing', () => {
    const rendered = TestRenderer.create(<BitHubLabLogo />);
    expect(rendered).toMatchSnapshot();
  });
});
