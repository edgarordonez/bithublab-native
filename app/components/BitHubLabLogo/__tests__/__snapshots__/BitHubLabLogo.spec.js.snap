// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BitHubLabLogo Renders without crashing 1`] = `
Array [
  <Image
    resizeMode="contain"
    source={1}
    style={
      Object {
        "height": 150,
        "width": 150,
      }
    }
  />,
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontFamily": "Roboto-Black",
        "fontSize": 50,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#2684ff",
        }
      }
    >
      Bit
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#000000",
        }
      }
    >
      Hub
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#8c929d",
        }
      }
    >
      Lab
    </Text>
  </Text>,
]
`;
